contracts/api-spec.yaml

```diff
@@ -269,18 +326,36 @@ components:
       description: A list of device serial numbers
       type: array
       items:
-        type: 'string'
-      example: ['BtmC8ckj']
+        $ref: '#/components/schemas/DeviceModel'
 
     DeviceModel:
       description: A device thats either an  encoder or decoder
       properties:
         serialNumber:
           type: string
           example: 'BtmC8ckj'
-        ipAddress:
+        publicIpAddress:
+          type: string
+          example: '178.156.52.3'
```

Comment:
This is a breaking change in the contract, please update the frontend to match.

Reply:
I added a new column in the device list page for the IP address.